docker/scripts/run.sh

```diff
@@ -0,0 +1,31 @@
+function build_docker_image {
+    BASENAME=$(basename $1);
+    IMAGE_NAME="newman_$BASENAME";
+    TAG="$2";
+    docker build -t "$DOCKER_ID_USER/$IMAGE_NAME:$TAG" .
+    if docker images | grep -q "$DOCKER_ID_USER/$IMAGE_NAME"; then
+        echo "Image built"
+        if docker run -t "$DOCKER_ID_USER/$IMAGE_NAME:$TAG" run "sample-collection.json"; then
```

Comment:
Don't you have to specify the complete path, as in `examples/sample-collection.json`?

Reply:
In this case, I already mounted the whole `examples` directory in `etc/newman`, so no need. The collection runs correctly.

Reply:
I did the mounting work in the dockerfile.